sw/device/tests/rv_core_ibex_address_translation_test.c

```diff
@@ -58,15 +58,19 @@ const dif_rv_core_ibex_addr_translation_slot_t kSlots[] = {
     kDifRvCoreIbexAddrTranslationSlot_0, kDifRvCoreIbexAddrTranslationSlot_1};
 
 // Translation descriptions to use.
+// Junk bits (0xDEADBEEF) are added to the `remap_addr` fields to ensure they
+// are ignored by the translation mechanism.
 static const dif_rv_core_ibex_addr_translation_mapping_t kMakeLowerCaseMapping =
     {
         .matching_addr = (uintptr_t)remapped_function,
-        .remap_addr = (uintptr_t)make_lower_case,
+        .remap_addr = (uintptr_t)make_lower_case +
+                      (uintptr_t)(0xDEADBEEF & (kRemapAlignment - 1)),
```

Comment:
I wonder if we should test both with and without junk bits?

We can drop the `const` on these variables, do one test without junk bits, add junk bits of the mapping in `test_main` and repeat the test.

Reply:
I'm not sure that really adds anything to the test

Reply:
@nbdd0121 happy to approve without this change?